fast-af/fast-checkout-woocommerce
# Add fast as payment gateway

# Description

This update includes Fast as a payment gateway in WooCommerce.  The payment gateway is disabled by default because it is not necessary for Fast to function properly. It is only being added for sellers who need Fast listed as a gateway in order to integrate with third-party systems.

# Testing instructions

1. Login to staging.
2. Verify that Fast is included as a payment option in the WooCommerce settings.

# Checks
